.github/workflows/tm2.yml

```diff
@@ -5,8 +5,6 @@ on:
     branches:
       - master
   pull_request:
-    paths:
-      - tm2/**
```

Comment:
Tm2 has no dependency on other parts of the codebase, so we can run it only on tm2 changes